third_party/proton/test/test_cmd.py

```diff
@@ -30,3 +30,15 @@ def test_exec(mode, tmp_path: pathlib.Path):
     kernels = data[0]["children"]
     assert len(kernels) == 2
     assert kernels[0]["frame"]["name"] == "test" or kernels[1]["frame"]["name"] == "test"
+
+
+def test_instrument_exec():
+
+    test_stderr = '0     matmul_kernel     instrument.py:43:20     SHARED     STORE\n'\
```

Comment:
Instead of comparing against the raw `stderr` output, you can probably process the output and store results into a table or a dict.